packages/patternfly-4/react-table/src/components/Table/Table.js

```diff
@@ -30,6 +30,8 @@ const propTypes = {
   className: PropTypes.string,
   /** Function called when user wants to collapse row. */
   onCollapse: PropTypes.func,
+  /** Function called when user wants to compound expand row. */
+  onExpand: PropTypes.func,
```

Comment:
Please make sure to add new properties to the d.ts file for table

Reply:
adding now...